routes.py

```diff
@@ -476,7 +515,10 @@ def createComment():
 def likeAPost():
```

Comment:
This function for the backend is still not working - changes are all good to debug but expect more to come as I try to help get it working